.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+***Status:***
+- [ ] ready
+- [ ] work in progress
+- [ ] hold
+
+
+**What is the purpose of this PR?**
+
+
+**Is this related to an Issue? (If so, please link to it)**
+
+
+**How should this be manually tested?**
+
+
+***Checklist:***
+- [ ] I have included appropriate Tests and have run `npm test` without errors
+- [ ] I have updated the Documentation accordingly
+- [ ] I updated typings/raven.d.ts if I added to or changed the API and have run `tsc --noEmit --noImplicitAny typescript/raven-tests.ts` without errors
```

Comment:
We can probably make this a script like `npm run test:ts`, or make it called by `npm test` directly